pkg/api/config_test.go

```diff
@@ -155,6 +155,64 @@ func TestValidateTests(t *testing.T) {
 			release:       &ReleaseTagConfiguration{Name: "origin-v3.11"},
 			expectedValid: true,
 		},
+		{
+			id: "invalid secret format",
+			tests: []TestStepConfiguration{
+				{
+					As:                                       "test",
+					OpenshiftAnsibleClusterTestConfiguration: &OpenshiftAnsibleClusterTestConfiguration{},
+					Secret: Secret{
+						Name: "secret",
+						Path: "/path/to/secret:exec",
```

Comment:
Is that not a valid filename on unix?

Reply:
It is valid. but in some mount operations (example in docker run this is how you pass in option) it defines that mountpoint should be executable. Better safe than sorry.